File: backend/src/models/kanban/JiraBoard/StatusSelf.ts
export class StatusSelf {
  self: string;
  name: string;
  untranslatedName: string;
  statusCategory: StatusCategory;

  constructor(
    self: string,
    name: string,
    untranslatedName: string,
    statusCategory: StatusCategory
  ) {
    this.self = self;
    this.name = name;
    this.untranslatedName = untranslatedName;
    this.statusCategory = statusCategory;
  }
}

export class StatusCategory {
  key: string;
  name: string;

  constructor(key: string, name: string) {
    this.key = key;
    this.name = name;
  }
}

File: backend/src/main/java/heartbeat/controller/board/vo/response/StatusSelf.java
package heartbeat.controller.board.vo.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.io.Serializable;

@AllArgsConstructor
@Data
@Builder
public class StatusSelf implements Serializable {

	private String untranslatedName;

	private StatusCategory statusCategory;

}
